cppwinrt/cppwinrt.vcxproj

```diff
@@ -115,7 +115,7 @@
     <RootNamespace>cppwinrt</RootNamespace>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
   </PropertyGroup>
-  <Import Project="$(SolutionDir)\cppwinrt.props" />
+  <Import Project="$(MSBuildThisFileDirectory)..\cppwinrt.props" />
```

Comment:
This is not the only vcxproj in the solution that refers to `$(SolutionDir)\cppwinrt.props`. Are the rest immune?

Reply:
Oops! You're right. I didn't check the test projects, as I wasn't planning on using those, and weren't building for me anyway (see my reply to your comment below).